scripts/run-in-pipeline.sh

```diff
@@ -65,6 +70,7 @@ skip_scan=${SKIP_SCAN:=$_skip_scan_default}
 fail_live=${FAIL_ON_LIVE:=$_fail_live_default}
 fail_unaudited=${FAIL_ON_UNAUDITED:=$_fail_unaudited_default}
 fail_audited_real=${FAIL_ON_AUDITED_REAL:=$_fail_audited_real_default}
+suppress_unscannable_file_warnings=${SUPPRESS_UNSCANNABLE_FILE_WARNINGS:=$_suppress_unscannable_file_warnings}
```

Comment:
I think you are missing "`_default`" here:
suppress_unscannable_file_warnings=${SUPPRESS_UNSCANNABLE_FILE_WARNINGS:=$_suppress_unscannable_file_warnings`_default`}

Reply:
good catch! https://github.com/IBM/detect-secrets/pull/121/commits/0a46536412219c64f4e603baa08cb6f13e9c624e